kv.go

```diff
@@ -399,26 +400,28 @@ func (js *js) CreateKeyValue(cfg *KeyValueConfig) (KeyValue, error) {
 		Duplicates:        duplicateWindow,
 		MaxMsgs:           -1,
 		MaxConsumers:      -1,
+		AllowDirect:       true,
```

Comment:
I think this should be an option as we discussed during the server PRs there are concerns around read-after-write when this is in use.  Generally read-after-write less important than we first thought so probably it will be fine, but we need to provide the option for users to retain current usage patterns

Reply:
Should not be an option imo from client perspective.